pkg/edit/builtins.go

```diff
@@ -17,16 +17,24 @@ import (
 
 //elvdoc:fn binding-table
 //
+// ```elvish
+// edit:binding-table $map
+// ```
+//
 // Converts a normal map into a binding map.
 
 //elvdoc:fn -dump-buf
 //
+// ```elvish
+// edit:-dump-buf
+// ```
+//
 // Dumps the current UI buffer as HTML. This command is used to generate
 // "ttyshots" on the [website](https://elv.sh).
 //
 // Example:
 //
-// ```elvish
+// ```elvish-transcript
```

Comment:
This is not a transcript; it probably should be changed to be one, but then you'd need to add prompts to the content.

Reply:
Actually, I'll just remove this command altogether now that your `ttyshot` change has been integrated.